extensions/integration-tests/src/notebook.test.ts

```diff
@@ -104,9 +104,49 @@ if (context.RunTest) {
 				assert(sparkResult === '2', `Expected spark result: 2, Actual: ${sparkResult}`);
 			});
 		}
+
+		test('scala language test', async function () {
```

Comment:
nit: These names could be improved - just by name they seem like they should be testing something about the actual language when all you're checking that the metadata in the file is parsed correctly. Would suggest something like 

'Notebook loads and parses scala language correctly'